block/src/Pos/Block/Network/Types.hs

```diff
@@ -71,3 +73,9 @@ data MsgBlock
     = MsgBlock Block
     | MsgNoBlock Text
     deriving (Eq, Show, Generic)
+
+-- | 'EncodedBlock' message
```

Comment:
Should be `'SerializedBlock'`?

Reply:
Indeed it should be `'SerializedBlock'`.